terraform/.gitignore

```
@@ -0,0 +1,4 @@
+# Terraform
+terraform.tfstate*
```

When I ran this locally, I got `main_local/local-cluster.tfstate` and `main_local/local-cluster.tfstate.backup`.
```suggestion
*.tfstate
*.tfstate.backup
```